lib/ui/channel_buffers.dart

```diff
@@ -185,6 +187,30 @@ class ChannelBuffers {
       await callback(message.data, message.callback);
     }
   }
+
+  String _getString(ByteData data) {
+    final ByteBuffer buffer = data.buffer;
+    final Uint8List list = buffer.asUint8List(data.offsetInBytes, data.lengthInBytes);
+    return utf8.decode(list);
+  }
+
+  /// Handle a control message.
+  ///
+  /// This is intended to be called by the platform messages dispatcher.
+  ///
+  /// Available messages:
+  /// - Name: resize
+  ///   Arity: 2
+  ///   Format: `resize\r<channel name>\r<new size>`
+  ///   Description: Allows you to set the size of a channel's buffer.
+  void handleMessage(ByteData data) {
```

Comment:
Is the structure of these control messages, name of the channel, etc. documented anywhere or is this a new convention?

Reply:
It's new.  I've documented them in the docstring.  If there is somewhere else you'd like to see them let me know.  The intention is that there is nice API's overtop of them so no one has to create them manually.